dashboard/components/navigation/blockchains-nav.tsx

```diff
@@ -0,0 +1,111 @@
+'use client'
+
+import { BLOCKCHAIN_LINKS } from '@/config/navigation'
+import { useMatchPath } from '@/lib/hooks/match-path'
+import { matchPath } from '@/lib/path'
+import { cn } from '@/lib/utils'
+import PolygonZkEvmSvg from '@/public/polygon-zk-evm-logo.svg'
+import ScrollSvg from '@/public/scroll-logo.svg'
+import ZkSyncEraSvg from '@/public/zk-sync-era-logo.svg'
+import { usePathname } from 'next/navigation'
+
+import { NavLink } from './navigation-link'
+
+interface BlockchainLink {
+  path: string
+  title: string
+  logo: JSX.Element
+}
+
+const navigationConfig = {
+  scroll: {
+    ...BLOCKCHAIN_LINKS.scroll,
+    logo: <ScrollSvg className="size-5" />,
+  },
+  zkSyncEra: {
+    ...BLOCKCHAIN_LINKS.zkSyncEra,
+    logo: <ZkSyncEraSvg className="size-5" />,
+  },
+  polygonZkEVM: {
+    ...BLOCKCHAIN_LINKS.polygonZkEVM,
+    logo: <PolygonZkEvmSvg className="size-5" />,
+  },
+}
```

Comment:
Changed to a more "extended" approach rather than duplicated. [97bcc6](https://github.com/MAYA-ZK/zk-dashboard/pull/22/commits/97bcc674805ed1d287aff93735ebe58ed4f924b4#diff-25ae5aff787937330951273416940d5c01c6c6efe57a89174f8a924fe6f2308bR18)